src/intro.md

```diff
@@ -67,57 +68,57 @@ Unified Extensible Firmware Interface (UEFI) and other firmware, particularly
 the network stack and file system modules, with Linux applications.
 
 LinuxBoot brings up the Linux kernel as a DXE in flash ROM instead of the UEFI
-shell. The Linux kernel, with a provided Go based userland, can then bring up
-the kernel that you want to run on the machine. The LinuxBoot firmware paradigm
-enables writing traditional firmware applications such as bootloader,
-debugging, diagnosis, and error detection applications as cross-architecture
-and cross-platform portable Linux applications.
+shell. The Linux kernel, with a provided Go based user-space, can then load the
+runtime kernel. The LinuxBoot paradigm enables writing traditional firmware
+applications such as boot loader, debugging, diagnosis, and error detection
+applications as cross-architecture and cross-platform portable Linux
+applications.
 
 When Linux boots it needs a root file system with utilities. One such root
-filesystem used for Linuxboot is based on u-root standard utilities written in
+filesystem used for LinuxBoot is based on u-root standard utilities written in
 Go. The following diagram shows the current state of the UEFI boot process and
 what is planned for the transition to LinuxBoot.
 
 [![comparison of UEFI boot and LinuxBoot](../images/UEFI-versus-LinuxBoot.svg)](../images/UEFI-versus-LinuxBoot.svg)
 
-## Benefits of using the Go userland environment and compiler
+## Benefits of using the Go user-space environment and compiler
 
 Go is a systems programming language created by Google. Go has strong typing,
 language level support for concurrency, inter-process communication via
 channels, runtime type safety and other protective measures, dynamic allocation
 and garbage collection, and closures. Go has a package syntax similar to Java
-that makes it easy to determine what packages a given program needs.
+that makes it easier to determine what packages a given program needs.
 
 The modern language constructs make Go a much safer language than C. This
 safety is critical for network-attached embedded systems, which usually have
 network utilities written in C, including web servers, network servers
-including sshd, and programs that provide access to a command interpreter,
+including `sshd`, and programs that provide access to a command interpreter,
 itself written in C. All are proving to be vulnerable to the attack-rich
 environment that the Internet has become.
 
-Even the most skilled programmers make simple mistakes that in C can be fatal,
-especially on network connected systems. Currently, even the lowest-level
-firmware in our PCs, printers, and thermostats is network-connected. These
-programming mistakes are either impossible to make in Go or, if made, are
-detected at runtime and result in the program exiting.
+Even the most skilled programmers can make mistakes that in C can be
```

Comment:
This changes the meaning. How about "can easily make mistakes"?

Reply:
See also `Using 'easy' may come across as condescending.` Vale error

Reply:
This is really subjective. And I think many would disagree with that broad of a statement, especially folks with many years of C programming experience. And what does "easy" mean? Classical C toolchains or poorly configured modern ones allow for unrecoverable bugs to occur? Better to link to a sources and perhaps phrase it like, "proponents of more modern safety focused languages argue that unrecoverable bugs are less likely".

Reply:
At the end of the day I'd just like to merge this rather than get into a language war. The whole thing needs to get rewritten anyway.

Reply:
yea gotcha, adding references to this discussion is for another day